ui/src/pages/backoffice/Item/ItemDetails/components/index.js

```diff
@@ -1,2 +1,4 @@
 export { ItemMetadata } from './ItemMetadata';
 export { ItemPastLoans } from './ItemPastLoans';
+export { ItemCirculation } from './ItemCirculation';
```

Comment:
Sort them if you want :)

Reply:
I don't think we anything to sort automatically in JS. Honestly, this is something we can ignore :)